aspnet/mobile/native-mobile-backend.rst

```diff
@@ -1,11 +1,170 @@
-.. include:: /../common/stub-topic.txt
+:version: 1.0.0
 
-.. _native-mobile-backend:
+Creating Backend Services for Native Mobile Applications
+========================================================
 
-|stub-icon| Creating Backend Services for Native Mobile Applications
-====================================================================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Mobile apps can easily communicate with ASP.NET Core backend services.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/mobile/native-mobile-backend/sample>`__
+
+The Sample App
+--------------
+
+This tutorial demonstrates how to create backend services using ASP.NET Core MVC to support native mobile apps. It uses the `Xamarin Forms TodoRest app <https://developer.xamarin.com/guides/xamarin-forms/web-services/consuming/rest/>`_ as its native client, which includes separate native clients for Android, iOS, Windows Universal, and Window Phone devices. The sample application includes an ASP.NET Web API 2 services project, which this article's ASP.NET Core MVC app replaces (with no changes required by the client).
+
+.. image:: native-mobile-backend/_static/todo-android.png
+
+.. note:: Securing web APIs is not covered in this article, but will be covered in follow-up documentation.
```

Comment:
I'd prefer not to have this note, so that we don't have to remember to remove it once the security content is available.